systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -304,4 +306,36 @@ public void testDeployKafkaOnPersistentStorage() {
         assertThat(configMap, valueOfCmEquals("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5"));
         assertThat(configMap, valueOfCmEquals("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
     }
-}
+
+    @Test
+    @KafkaCluster(name = "my-cluster", kafkaNodes = 1, zkNodes = 2, config = {
+            @CmData(key = "zookeeper-healthcheck-delay", value = "30"),
+            @CmData(key = "zookeeper-healthcheck-timeout", value = "10"),
+            @CmData(key = "kafka-healthcheck-delay", value = "30"),
+            @CmData(key = "kafka-healthcheck-timeout", value = "10"),
+            @CmData(key = "KAFKA_DEFAULT_REPLICATION_FACTOR", value = "2"),
+            @CmData(key = "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", value = "5"),
+            @CmData(key = "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", value = "5")
+    })
+    @OpenShiftOnly
+    public void testForUpdateValuesInConfigMap() {
+        String clusterName = "my-cluster";
+        int expectedZKPods = 2;
+        Oc oc = (Oc) this.kubeClient;
+
+        replaceCm(clusterName, "zookeeper-healthcheck-delay", "23");
+        oc.waitForStatefulSet(zookeeperStatefulSetName(clusterName), expectedZKPods);
+
+        String configMap = kubeClient.get("cm", clusterName);
+        assertThat(configMap, valueOfCmEquals("zookeeper-healthcheck-delay", "23"));
+        assertThat(configMap, valueOfCmEquals("zookeeper-healthcheck-timeout", "10"));
+
+        List<String> ccPods = oc.listByLabel("pod", "strimzi-cluster-controller");
+        LOGGER.info("Count of Cluster controllers is: " + ccPods.size());
+        for (int i = 0; i < ccPods.size(); i++) {
+            Assert.assertThat(oc.logs(ccPods.get(i)),
+                    CoreMatchers.containsString("Diff: Zookeeper healthcheck timing changed"));
```

Comment:
Relying on the log seems like something what can be quite unstable. Would it be easier to get the pods from the StatefulSet and check the value of the healtcheck delay which you changed?

Reply:
@scholzj It's a good point but could you please explain how to get this value directly from the pod?

Reply:
Is it possible to do that using some command?

Reply:
@asorokhtey Well, it is in the JSON you get from the `oc get pod -o json` command - in the sections `livenessProbe` and `readinessProbe`.